frontend/src/pages/Login.jsx

```diff
@@ -36,20 +38,31 @@ function Login() {
           });
       }}
     >
+      <div className="PresInscip">
+        <img src={Logo} alt="Ino Vin" />
+        <h1>CONNEXION</h1>
+      </div>
       <div>
-        <label htmlFor="username">Username</label>
-        <input ref={usernameRef} type="text" id="username" name="username" />
+        <input
```

Comment:
inputs should have labels, for a11y purposes, and placeholders should present value example, not label text

Reply:
are you looking for this kind of effect ?